src/App.js

```diff
@@ -1,12 +1,51 @@
-import React from 'react';
-import './App.css';
+import {BrowserRouter as Router, Switch, Route} from "react-router-dom";
+import './App.css'
+import Pokemon from "./Components/Pokemon";
+import {useEffect, useState} from "react";
+import axios from "axios";
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+
+    const [pokemons, setPokemons] = useState(null);
+    const [endpoint, setEndpoint] = useState(`https://pokeapi.co/api/v2/pokemon`);
+
+    useEffect(() => {
+        async function fetchData() {
+            try {
+                const result = await axios.get(`${endpoint}`);
+                console.log(result.data)
+                setPokemons(result.data)
+            } catch (e) {
+                console.error(e);
+            }
+        }
+
+        fetchData();
+
+    }, [endpoint]);
+
+    return (
+
+        <div>
+            <button
+                type='button'
+                onClick={()=> setEndpoint(pokemons.next)}
```

Comment:
`.next` kun je in beginsel niet gebruiken, want `pokemons` is null. Ik zou een extra check toevoegen:
```
if (pokemons === null) return null;
```